### PR TITLE
Create the javascript enabled details popup window

### PR DESCRIPTION
**Pull request by codepantha**
## Changes Introduced into the project are as follows: 👇 
- Design and style the details popup window
- Make the details popup modal responsive
- Dynamically load the work/projects section
- Display the project details modal with the relevant details using javascript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -254,7 +177,33 @@ <h2>Skills</h2>
...
+      <div class="innerDiv">
+        <button class="close-modal-button">X</button>
+        <div class="modal-card">
+          <img src="" alt="smartfacedetector" class="modal-img" id="modal-img"/>
+          <div class="details-without-img">
+            <div>
+              <h4 class="title-modal-align" id="modal-title"></h4>
+            </div>
+            <ul class="projectDetailsModalTechGroup" id="modal-tech">
+              <!-- <li class="techModal">Nodejs</li>
```
Comment (carlos-ssh) [inline index.html]: - Remove commented code.

File: index.html [hunk 2]
```diff
@@ -254,7 +177,33 @@ <h2>Skills</h2>
...
+              <li class="techModal">css</li>
+              <li class="techModal">Javascript</li>
+              <li class="techModal">html</li> -->
+            </ul>
+            <div class="project-description" id="modal-desc">
+              <!-- Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea -->
```
Comment (carlos-ssh) [inline index.html]: - Remove commented code.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Code-panther , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (codepantha) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (codepantha) [reply index.html]: Fixed.

**Review by carlos-ssh — APPROVED** [review]
Hi @Code-panther , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**